components/org.wso2.carbon.status.dashboard.web/src/gadgets/chart-components/PieChart.jsx

```diff
@@ -221,28 +199,28 @@ export default class PieCharts extends React.Component {
                                     return [
                                         {
                                             target: 'data',
-                                            mutation: this._handleMouseEvent
-                                        }
+                                            mutation: this._handleMouseEvent,
```

Comment:
comma not needed..

Reply:
check other such places

Reply:
Fixed in bbb066d61f625e5e28b09b91ec2e5a2dc09ed919 .Move to separate repo.